ProcessMaker/processmaker
# FOUR-12175 | Develop ‘Connected Accounts’ Tab Under User Settings

# Feature
Ticket: [FOUR-12175](https://processmaker.atlassian.net/browse/FOUR-12175)

This PR introduces the initial iteration of the Connected Accounts user setting as part of the ongoing development for the Wizard Templates feature. The Connected Accounts section is designed to display a list of external accounts that are available for a user to connect to their profile. Users are also able to edit an account connection from this screen.

# How to Test
1. Go to branch `task/FOUR-12175` in `processmaker`.
2. Login to ProcessMaker. Click on the Avatar Image in the upper right corner of the screen.
3. Select the 'Edit User Profile' option.
4. You should be navigated to the Edit Profile page. Select the Connected Accounts tab.
	- This tab should display accounts. (Please note that the accounts displayed are dummy data that will be updated.)
5. When the 'Edit' button is clicked for an account, a modal to Edit Connection should appear.

ci:next

# Code Review Checklist


[FOUR-12175]: https://processmaker.atlassian.net/browse/FOUR-12175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
